avae/train.py

```diff
@@ -77,8 +82,22 @@ def train(
         params=vae.parameters(), lr=learning  # , weight_decay=1e-5
     )
 
+    beta_arr = KLD_weight(
+        epochs,
```

Comment:
Since this is loss related, does it make more sense to call this in the constructor of the loss?